crates/swc_ecma_transforms_base/src/helpers/_apply_decorated_descriptor.js

```diff
@@ -16,14 +16,6 @@ function _applyDecoratedDescriptor(target, property, decorators, descriptor, con
     desc.value = desc.initializer ? desc.initializer.call(context) : void 0;
     desc.initializer = undefined;
   }
-
-  var own = Object.getOwnPropertyDescriptor(target, property);
```

Comment:
Please apply the same things to helpers in https://github.com/swc-project/swc/tree/02c401830d49632e5fbea7280ca703d27c3917d9/packages/swc-helpers

Reply:
Oh, I see. 🤣